test/fixtures/dialects/postgres/l042_issue_2898_redshift_attribute_error.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO target_table (target_column)
```

Comment:
I think we should rename this file from:
```
l042_issue_2898_redshift_attribute_error.sql
```
to
```
postgres_cte.sql
```

Reply:
Ah, good point.

Reply:
Done!

Reply:
Actually it’s not a CTE, it’s a nested table. Should we rename again? Sorry!

Reply:
Renamed to `postgres_subquery.sql`.